README.md

```diff
@@ -1 +1,183 @@
-# kanban-rod-haben
\ No newline at end of file
+
+<div align="center">
+
+  <h2><b>Awesome-Series</b></h2>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠️ Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 [Awesome-Series] 
+
+This is an application allows users to browse active tv shows from tvShow Api. It also allows them to comment on shows and see what others say about the show.
+
+   ### [Video](https://drive.google.com/file/d/1pqZ3MZCuONYJt4kgM12R7zsJrBi0Syyl/view?usp=sharing)
+
+## 🛠️ Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://html.org/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://css.com/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>JavaScript</summary>
+  <ul>
+    <li><a href="https://jacascript.com/">JavaScript</a></li>

```

Comment:
- [ ] Kindly fix the broken links (HTMLd JavaScript), implementing this will make your repo to be more engaging. 👍